cmd/clusterctl/cmd/version_checker.go

```diff
@@ -61,7 +64,12 @@ func newVersionChecker(ctx context.Context, vc config.VariablesClient) (*version
 		tc := oauth2.NewClient(ctx, ts)
 		client = github.NewClient(tc)
 	} else {
-		client = github.NewClient(nil)
+		githubClient = github.NewClient(nil)
```

Comment:
```suggestion
		client = github.NewClient(nil)
```
It's not compiling since `githubClient` is not defined as a variable, I think you meant to rename it to `client`